.vscode/settings.json

```diff
@@ -1,18 +1,26 @@
 {
-    "files.trimTrailingWhitespace": true,
-    "editor.tabSize": 4,
```

Comment:
Why remove remove the whitespace indention setting?

Reply:
`"editor.insertSpaces": true` and `"editor.tabSize": 4` are VSCode defaults.
Besides, with `"editor.detectIndentation": true` (also VSCode default), `"editor.tabSize"` and `"editor.insertSpaces"` are automatically detected.

So they would be redundant.

Of course, one can make a case for the situation where a developer might have changed these settings in their local editor config. But in that case, we would have to apply project overrides for far too many settings to compensate for the developer's "creativity" with local settings 🙂. Such situations are usually rare though.